packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -587,6 +589,17 @@ async function setupOrgBrowser(
 }
 
 export async function activate(extensionContext: vscode.ExtensionContext) {
+  // Set the value of ext.supportedFolders by reading the packages that are
+  // in the sfdx-project.json file.  This property is used by the
+  // Generate Manifest File command, which should be available under all dirs
+  // listed as packages in the project config file
+  const project = SfProject.getInstance(getRootWorkspacePath());
+  const projectDirectories = project.getUniquePackageNames();
```

Comment:
Function ref:
https://github.com/forcedotcom/sfdx-core/blob/df2c945b47d3c79fd1c2e32f6fa35b18007a9cda/src/sfProject.ts#L594

Reply:
Could we add a method to configUtils in vscode-utils to return the project directories? Since it's reading the config file configUtils seems like a logical location.

Reply:
And then unit testing that method would be a lot simpler :)